resume/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Section from "./components/Section";
+import resumeJSON from "./data/resume";
+import Basics from "./components/sections/Basics";
+import Work from "./components/sections/Work";
```

Comment:
Similar to the comment I made on David's that I liked the use of components. Looks like you took it a step further and stored them in their own location. Nice!